sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -2005,7 +2013,7 @@ public void processElement(
                       KV.of("hello", 42),
                       KV.of("hello", 84),
                       KV.of("hello", 12)))
-              .apply(ParDo.of(fn).withSideInputs(listView));
```

Comment:
As a part of test case, we need to remove `withSideInputs` and dynamically add in ParDo